website/client/src/components/groups/groupFormModal.vue

```diff
@@ -115,7 +115,8 @@ label.custom-control-label(v-once) {{ $t('allowGuildInvitationsFromNonMembers')
       </div>
       <div class="form-group">
         <label>
-          <strong v-once>{{ $t('groupDescription') }} *</strong>
+          <strong v-if="isParty" v-once>{{ $t('groupDescription') }}</strong>
+          <strong v-if="!isParty" v-once>{{ $t('groupDescription') }} *</strong>
```

Comment:
could we use a `v-else` for the second condition? We should also remove the `v-once`(s) since they prevent rendering based on a condition

Reply:
Done!